operations/research-and-development/README.md

```diff
@@ -31,7 +57,7 @@ R&D includes engineering, QA, product management, and design \(UX, UI\).
 * Eric Sethna - Product Manager
 * Aaron Rothschild - Product Manager
 
-#### Technical Writers
+#### Technical Writer
```

Comment:
Why isn't Justine Geffen listed here? :)

Reply:
Because this is the messaging vertical and there are overarching PMs and TW. For the other verticals, they're individual so this structure wouldn't work. Not sure if that makes sense?

Reply:
Aha! This content is specific to the Messaging vertical. Got it!

Reply:
You question does alert me to the fact that it's not immediately obvious that we have verticals.